include/picolibrary/crc.h

```diff
@@ -330,6 +330,11 @@ class Calculator_Concept {
 /**
  * \brief Bitwise calculator.
  *
+ * This calculator implementation processes messages a single bit at a time, and requires
+ * a message augment to push the entirety of the message through the calculation. While
+ * this results in lower performance than table driven implementations, memory use is
+ * lower due to the lack of a lookup table.
+ *
```

Comment:
The documentation for the calculator implementations are not consistent and require corrections.